stories/DropdownMenuItemButtonCheckable-stories.tsx

```diff
@@ -0,0 +1,292 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import styled from '@emotion/styled';
+import type { Meta, StoryObj } from '@storybook/react';
+import { PropsWithChildren, forwardRef } from 'react';
+import { DropdownMenu, IconChevronDown } from '../src';
+
+const meta: Meta<typeof DropdownMenu.ItemButtonCheckable> = {
+  component: DropdownMenu.ItemButton,
+  title: 'DropdownMenu.ItemButtonCheckable',
```

Comment:
You could replace the dot with a slash in the title. This will create a kind of category in storybook:
![image](https://github.com/SonarSource/echoes-react/assets/47277647/d58e9a28-59e7-4085-881c-739dccf626b8)